src/super_gradients/training/datasets/detection_datasets/detection_dataset.py

```diff
@@ -449,6 +449,18 @@ def output_target_format(self):
                 target_format = transform.output_format
         return target_format
 
+    @staticmethod
+    def _standardize_image(image):
+        # Normalize the image to have minimum of 0 and maximum of 1
+        image_min = image.min()
+        image_max = image.max()
+        normalized_image = (image - image_min) / (image_max - image_min)
```

Comment:
(max - min + eps) to avoid division by zero if image is happen to be a be flat

Reply:
done